main/src/cgeo/geocaching/connector/gc/GCWebAPI.java

```diff
@@ -232,6 +233,13 @@ String getAuthorizationField() {
         @JsonProperty("longitude")
         double longitude;
 
+        PostedCoordinates(final double latitude, final double longitude) {
+            this.latitude = latitude;
+            this.longitude = longitude;
+        }
+
+        PostedCoordinates() {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['}' is not preceded with whitespace.](https://app.codacy.com/manual/cgeo/cgeo/pullRequest?prid=5262679)